static/app/components/events/interfaces/performance/spanEvidence.tsx

```diff
@@ -1,9 +1,9 @@
 import styled from '@emotion/styled';
 
+import {EventDataSection} from 'sentry/components/events/eventDataSection';
 import {t} from 'sentry/locale';
 import {EventTransaction, IssueType, Organization} from 'sentry/types';
 
-import {DataSection} from '../../eventTagsAndScreenshot/dataSection';
```

Comment:
Having two very similarly named things that kind of do the same thing is dangerous 😬